src/main/scala/scalty/types/or.scala

```diff
@@ -71,12 +72,12 @@ object OrTypeExtensions {
       * @return [[scalty.types.OrTypeAlias.EmptyOr]]
       */
     def recoverToEmptyOr(implicit ec: ExecutionContext): EmptyOr =
-      XorT.apply(or.value.map(_ => xor.EMPTY_XOR))
+      or.value.map(_ => xor.EMPTY_XOR).toEmptyOr
```

Comment:
It's convertation `Either[AppError, T]` to empty
```scala
map(_ => xor.EMPTY_XOR)
```
And then more one convetation
```scala
.toEmptyOr
```

Maybe it would be better write like
```scala
or.value.toEmptyOr
```